angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.component.ts

```diff
@@ -180,7 +180,7 @@ import {
         >
           <div
             *ngIf="narrowSelector['footer']"
-            class="sprk-c-Dropdown__footer sprk-u-TextAlign--center"
+            class="sprk-c-Masthead__selector-footer sprk-u-TextAlign--center"
```

Comment:
Since we have our own class now, we will want to remove this center utility that applies `!important` CSS and add text align center to our own class.

Reply:
Hey @RVMendoza, looks like you missed this comment from last review.